src/pages/LoginAndRegister/RegisterSocial.vue

```diff
@@ -221,7 +262,28 @@ export default {
 				this.$kvTrackEvent('Register', 'error-register-social-form-invalid-input');
 			}
 		},
+		fetchStrategicPartnerLoginInfoByPageId() {
+			this.apollo.query({
+				query: StrategicPartnerLoginInfoByPageId,
+				variables: { pageId: this.partnerContentId }
+			}).then(res => {
+				const spLoginInfo = res?.data?.strategicPartnerLoginInfoByPageId;
+				const logo = spLoginInfo?.contentful?.entry?.fields?.primaryLogo;
+				this.fetchedLogo = {
+					url: logo?.fields?.file?.url || '',
+					alt: logo?.fields.title || '',
```

Comment:
Does this need to be `logo?.fields?.title`?

Reply:
done